snapshot_iterator_integration_test.go

```diff
@@ -377,3 +377,91 @@ func TestSnapshotIterator_DeleteEndWhileSnapshotting(t *testing.T) {
 		testutils.AssertUserSnapshot(is, users[i-1], rec)
 	}
 }
+
+func TestSnapshotIterator_StringSorting(t *testing.T) {
+	// MySQL sorts strings in a case-insensitive manner, so we need to test that we
```

Comment:
I like that the comment is explaining what the test is about, but we

need to finish the sentence.:)

Reply:
oh damn ahahahah, good catch, I was multitasking way too much